README.md

```diff
@@ -1 +1,153 @@
-# Vet-clinic-DB
\ No newline at end of file
+# Clinic Record
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Clinic Records <a name="about-project"></a>
+
+Database Creation based on Schema Diagram is a project where a database has been created from the given schema Diagram.
+
+**Clinic Record** is a is a project showing schema diagram for a clinic. The database has to be created making SQLcommands.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+Schema Diagram
+
+Creation of Database
+
+- **Idea about the Schema Diagram**
+- **How to understand different Tales with their key features**
+- **Execution of SQL commands to construct the required Database**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+This repository includes 1 file named schema_based_on_diagram.sql with SQL commands which will be used to create the required database.
+
+### Prerequisites
+
+In order to run this project you need: PgAdmin4 GUI
+
+### Setup
+
+Clone this repository to your desired folder.
+
```

Comment:
- [ **OPTIONAL** ] **IMPORTANT:** I think it would be better if you it would be better if you also add some command of how to `clone` your project, `access` the cloned directory to help non technical users to know how to proceed.